src/features/projects/components/ProjectSnippet.tsx

```diff
@@ -41,7 +41,7 @@ export default function ProjectSnippet({
   const { token } = React.useContext(UserPropsContext);
   const handleOpen = () => {
     const url = getDonationUrl(project.slug, token);
-    window.location.href = url;
+    window.open(url, '_blank').focus();
```

Comment:
@prachigarg19 This should be done only while embed=true. Otherwise, it should remain as previous.